src/Commands/RunCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nitishkumar
+ * Date: 3/8/18
+ * Time: 9:51 AM
+ */
+
+namespace Hyn\Tenancy\Commands;
+
+use Hyn\Tenancy\Contracts\Repositories\WebsiteRepository;
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Console\Kernel;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Hyn\Tenancy\Models\Website;
+
+class RunCommand extends Command
+{
+    protected $signature = 'tenancy:run {tenantcommand}
```

Comment:
Maybe change `{tenantcommand}` to `{command}` - I think it is clear what the argument stands for.